Localazy/Service/LocalazyFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Localazy.Model;
+
+namespace Localazy.Service;
+
+public class LocalazyFactory : ILocalazyFactory
+{
```

Comment:
Instead of creating a new `HttpClient`, we should proably use the `IHttpClientFactory`.
```suggestion
{
    private readonly IHttpClientFactory _httpClientFactory;

    public LocalazyFactory(IHttpClientFactory httpClientFactory)
    {
        _httpClientFactory = httpClientFactory;
    }

```